submariner/multiple-cable-drivers.md

```diff
@@ -0,0 +1,415 @@
+
+The `ClusterConnectionPolicies` resources would be created and maintained on the broker cluster and synced to each participating cluster.
+A controller to reconcile updates to policies will need to be implemented in Submariner.
+
+### Labelling clusters
+
+Clusters are defined as Custom Resources and can be labeled in the CR definition, or after creation using kubectl. A new parameter to label
+clusters at `subctl join` time could also be added for usability purposes.
+
+### Policy management with subctl
+
+The `ClusterConnectionPolicies` can be created as the Broker deployment through the extension of submariner configuration YAML to include
+parameters that support policy creation.
+
+The `ClusterConnectionPolicies` can be created/managed after Broker deployment through kubectl or subctl using a command such as:
+
+```cmd
+subctl connection-policy add | list | delete | export
+```
+
+> **_NOTE:_** If a policy CR is created by a cluster admin using kubectl - it will need to be synced to the Broker.
+
+#### subctl connection-policy commands
+
+##### `add`
+
+`subctl connection-policy add [flags]`
+
+<!-- markdownlint-disable line-length -->
+| Flag                                  | Description
+|:--------------------------------------|:---------------------------------------------------------------------------------------------------|
+| `--kubeconfig` `<string>`             | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
+| `--kubecontext` `<string>`            | Kubeconfig context to use
+| `--name` `<string>`                   | The name of this policy (used in the metadata)
+| `--left-cluster-selector` `<string>`  | Comma separated list of cluster labels for the label selector to use
+| `--right-cluster-selector` `<string>` | Comma separated list of cluster labels for the label selector to use
+| `--cluster-selector` `<string>`       | Comma separated list of cluster labels for the label selector to use
+| `--topology` `<string>`               | The topology to use: full-mesh/point-to-point/client-server
+| `--priority` `<value>`                | The priority of this policy
+| `--connection-policy` `<string>`      | The connection policy: connect/no-connect
+| `--cable-driver` `<string>`           | The name of the cable driver to use for this policy
+| `--cable-config`  `<string>`          | The cable driver ConfigMap name
+<!-- markdownlint-enable line-length -->
+
+##### `delete`
+
+`subctl connection-policy delete [flags]`
+
+| Flag                       | Description
+|:---------------------------|:-------------------------------------------------------------------------------|
+| `--kubeconfig` `<string>`  | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
+| `--kubecontext` `<string>` | Kubeconfig context to use
+| `--name` `<string>`        | The name of the policy to delete
+
+##### `export`
+
+`subctl connection-policy export [flags]`
```

Comment:
What does it mean to "export" it?

Reply:
The intention was to create the connectivity policy locally on the cluster and then export it in a similar way we export services (use a federator to propagate it to the Broker)...

Reply:
Connectivity policies need to be automatically exported/synced to the broker, right? Each cluster needs to observe the same connectivity policies.